s2p/rpc_utils.py

```diff
@@ -279,7 +279,8 @@ def utm_zone(rpc, x, y, w, h):
     return geographiclib.compute_utm_zone(lon, lat)
 
 
-def roi_process(rpc, ll_poly, use_srtm=False):
+def roi_process(rpc, ll_poly, use_srtm=False, exogenous_dem=None,
+                exogenous_dem_geoid_mode=True):
```

Comment:
The parameters have to be passed explicitly to the function and cannot be read from `cfg` as it is done in other functions, because when `roi_process()` is called, `cfg` has not been updated with the user's configuration options yet.